modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/display/context/BaseDLEditFileEntryDisplayContext.java

```diff
@@ -73,6 +73,13 @@ public BaseDLEditFileEntryDisplayContext(
 		}
 	}
 
+	@Override
+	public DDMFormValues getDDMFormValues(DDMStructure ddmStructure)
+		throws PortalException {
+
+		return parentDisplayContext.getDDMFormValues(ddmStructure);
+	}
+
 	@Override
 	public DDMFormValues getDDMFormValues(long classPK) throws PortalException {
```

Comment:
Can you check if this method is no longer used? In that case, remove it.

Reply:
Because the interface that it implements was added an abstract method.